### PR TITLE
Seeding Complete

### PR DESCRIPTION
**Pull request by karebae**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const database = require('../database/index.js');
+const seed = require('../database/seeding.js');
+
+const app = express();
+const port = 3000;
+
+app.listen(port, () => {
+  console.log(`Karin's app listening at http://localhost:${port}`)
```
Comment (jonfu86) [inline server/index.js:9]: a nice touch

File: database/seeding.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      quality_rating: faker.random.number({ 'min': 1, 'max': 5 }),
+      value_rating: faker.random.number({ 'min': 1, 'max': 5 }),
+      star_rating: faker.random.number({ 'min': 1, 'max': 5 }),
+      helpful_yes: faker.random.number({ 'min': 1, 'max': 1000 }),
+      helpful_no: faker.random.number({ 'min': 1, 'max': 500})
+    };
+    hugeSeedingArray.push(seedling);
+  }
+  database.save(hugeSeedingArray)
+  .then(() => {
```
Comment (jonfu86) [inline database/seeding.js:37]: very clever

File: database/seeding.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  ----------------------------------------------------
+*/
+
+const faker = require('faker');
+var database = require('./index.js');
+//const mongoose = require('mongoose');
+
+(function seeding (){
+  // creating 100 entries into our database, all of them are called seedling
+  var hugeSeedingArray = [];
```
Comment (jonfu86) [inline database/seeding.js:16]: nice variable name

**Review by jonfu86 — APPROVED** [review]

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
```
Comment (rickboot) [inline webpack.config.js:1]: Nice work from the webpack master.

**Comment by rickboot** [comment]
Looks great to me!

**Pull request closed without merging**